AllHitAnalyze_New.py

```
@@ -1,162 +1,162 @@
+		matplotlib.pyplot.rcParams["font.sans-serif"] = ["LXGW WenKai Mono"]
+		grid:GridSpec = GridSpec(3, 5, left=0.045, right=1, top=1, bottom=0.06, wspace=0, hspace=0);
+		# create Bar Chart Axes
+		BarAxes:Axes = fig.add_subplot(grid[:,:]);
+		self.BarChart(BarAxes);
+		if self.config['EnableDonutChartinAllHitAnalyze']:
+			# create Ring Chart Axes
+			RingAxes:Axes = fig.add_subplot(grid[0:2,3:]);
+			RingAxes.add_patch(matplotlib.patches.Rectangle((-1.5, -1.5), 3, 3, color="white"));
+			self.RingChart(RingAxes);
+		# Show Plots
+		matplotlib.plt.show();
```

Ruff (0.14.8)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)